internal/cluster/distribution/eks/image_selector_defaults.go

```diff
@@ -65,78 +65,78 @@ var defaultImages = ImageSelectors{
 	KubernetesVersionImageSelector{ // Kubernetes Version 1.15.11
 		Constraint: mustConstraint("1.15"),
 		ImageSelector: RegionMapImageSelector{
-			"ap-east-1":      "ami-0ae1b0aad4d6fb508", // Asia Pacific (Hong Kong).
-			"ap-northeast-1": "ami-026e39e61d44ff507", // Asia Pacific (Tokyo).
-			"ap-northeast-2": "ami-0e1e660d5e393d5f1", // Asia Pacific (Seoul).
-			"ap-southeast-1": "ami-0d32de2029a9f56fd", // Asia Pacific (Mumbai).
-			"ap-southeast-2": "ami-07aee5ce871a45bcf", // Asia Pacific (Singapore).
-			"ap-south-1":     "ami-040e5afd1b110a399", // Asia Pacific (Sydney).
-			"ca-central-1":   "ami-0286b3b75d600924d", // Canada (Central).
-			"eu-central-1":   "ami-02497bca9c9dbc206", // EU (Frankfurt).
-			"eu-north-1":     "ami-0efc0441f778f5826", // EU (Stockholm).
-			"eu-west-1":      "ami-023736532608ff45e", // EU (Ireland).
-			"eu-west-2":      "ami-0a79663bf395ae44d", // EU (London).
-			"eu-west-3":      "ami-0de67e5d090c0eef0", // EU (Paris).
-			"me-south-1":     "ami-075a74dc065b91bf6", // Middle East (Bahrain).
-			"sa-east-1":      "ami-07b176b7f55c9df7c", // South America (Sao Paulo).
-			"us-east-1":      "ami-06d4f570358b1b626", // US East (N. Virginia).
-			"us-east-2":      "ami-0c1bd9eca9c869a0d", // US East (Ohio).
-			"us-west-1":      "ami-03ac9a33b2d6c61f7", // US West (N. California).
-			"us-west-2":      "ami-065418523a44331e5", // US West (Oregon).
+			"ap-east-1":      "ami-06c4a53520070412d", // Asia Pacific (Hong Kong).
+			"ap-northeast-1": "ami-0e263d94d831d6e3f", // Asia Pacific (Tokyo).
+			"ap-northeast-2": "ami-0696aab7814e872d5", // Asia Pacific (Seoul).
+			"ap-southeast-1": "ami-07ec30950a8cf5f5e", // Asia Pacific (Mumbai).
+			"ap-southeast-2": "ami-0163006b79677185b", // Asia Pacific (Singapore).
+			"ap-south-1":     "ami-0fd5657d22dd97c7a", // Asia Pacific (Sydney).
+			"ca-central-1":   "ami-032072399f84866fa", // Canada (Central).
+			"eu-central-1":   "ami-05acf7139b3fa4195", // EU (Frankfurt).
+			"eu-north-1":     "ami-0cc9a5fbe0fb4846f", // EU (Stockholm).
+			"eu-west-1":      "ami-0b4cbc24e98bbe268", // EU (Ireland).
+			"eu-west-2":      "ami-051e5ec4ed42120bf", // EU (London).
+			"eu-west-3":      "ami-0fced5d71992f332d", // EU (Paris).
+			"me-south-1":     "ami-0b55422936e1febca", // Middle East (Bahrain).
+			"sa-east-1":      "ami-0c882223525ac33e9", // South America (Sao Paulo).
+			"us-east-1":      "ami-055e79c5dcb596625", // US East (N. Virginia).
+			"us-east-2":      "ami-03c1ef6e2dcef9091", // US East (Ohio).
+			"us-west-1":      "ami-07d781add09539e02", // US West (N. California).
+			"us-west-2":      "ami-0b4f1df0761911a2a", // US West (Oregon).
 
 			// AWS GovCloud (US) partition
 			"us-gov-east-1": "ami-0a6bce000f1b3d6c3", // GovCloud (US-East)
```

Comment:
don't we have to update these too?

Reply:
This was the result of the `aws ssm get-parameter` command